modules/administration/pages/troubleshooting/tshoot-packages-proxy.adoc

```diff
@@ -0,0 +1,61 @@
+[[troubleshooting-repos-proxy]]
+= Troubleshooting Repository Via Proxy Issues
+
+////
+
+PUT THIS COMMENT AT THE TOP OF TROUBLESHOOTING SECTIONS
+
+Troubleshooting format:
+
+One sentence each:
+Cause: What created the problem?
+Consequence: What does the user see when this happens?
+Fix: What can the user do to fix this problem?
+Result: What happens after the user has completed the fix?
+
+If more detailed instructions are required, put them in a "Resolving" procedure:
+.Procedure: Resolving Widget Wobbles
+. First step
+. Another step
+. Last step
+
+////
+
+In some occasions the [systemitem]``squid`` cache on the proxy gets corrupted.
+When this occurs getting packages or repositories metadata on a client connected to the proxy fails with various possible error messages.
+
+Cleaning the [systemitem]``squid`` cache is done differently on a regular or a container proxy.
+
+For a regular proxy, follow this procedure on the proxy machine:
+
+----
+systemctl stop squid
+rm -rf /var/cache/squid
+systemctl start squid
+----
+
+For a container proxy running with [systemitem]``podman``, follow this procedure on the host machine:
+
+----
+systemctl stop uyuni-proxy-pod
+podman volume rm uyuni-proxy-squid-cache
+systemctl start uyuni-proxy-pod
+----
+
+For a container proxy running in a [systemitem]``Kubernetes`` cluster, follow this procedure:
+
+----
+kubectl delete deployment/uyuni-proxy
+PV_NAME=$(kubectl get pv -A | grep squid-cache-pv-claim | cut -f 1 -d ' ')
+kubectl patch pv $PV_NAME -p '{"metadata":{"finalizers": []}}' --type=merge
+kubectl delete pv/$PV_NAME
+----
+
+Deploy the pod again from the helm chart with the following command:
+
+----
+helm upgrade --install oci://...
+----
+
+Mind that this command uses `upgrade --install`.
```

Comment:
We usually use the simple expression "note ...".

```suggestion
Note that this command uses [command]``upgrade --install``.
```

Reply:
@cbosdo can you please also add a `.changelog` entry?

Reply:
Done

Reply:
thank you!